src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -1365,6 +1365,68 @@ public IActionResult SyncFromDate([FromBody] WalletSyncFromDateRequest request)
             return this.Ok();
         }
 
+        [Route("wallet-stats")]
+        [HttpGet]
+        public IActionResult WalletStats([FromQuery] WalletStatsRequest request)
```

Comment:
General question, what's the purpose of this PR is it mainly informational or is it needed by exchanges for example or maybe the test tool?

Reply:
This API provides information about the composition of UTXO within the wallet. It is a very useful functionality during testing but could prove helpful while debugging production issues.